connection.go

```diff
@@ -204,6 +204,14 @@ func (c *Connection) Init() error {
 		initOp.Flags |= fusekernel.InitAtomicTrunc
 	}
 
+	if !c.cfg.DisableReaddirplus {
+		// Enable Readdirplus support, allowing the kernel to use Readdirplus
+		initOp.Flags |= fusekernel.InitDoReaddirplus
+
+		// Enable adaptive Readdirplus, allowing the kernel to choose between Readdirplus and Readdir
+		initOp.Flags |= fusekernel.InitReaddirplusAuto
```

Comment:
I feel, we should expose ReaddirplusAuto as another flag, there could be reason that's why kernel exposed it as another flag.